config/constant.go

```diff
@@ -19,8 +19,9 @@ const (
 	DefaultPfMarkBootstrappedPath                 = "api/v2/node/containers/mark_bootstrapped"
 	DefaultPfMarkBootstrapErrorPath               = "api/v2/node/containers/mark_bootstrap_error"
 	DefaultPfMarkDeletedPath                      = "api/v1/node/containers/mark_deleted"
-	DefaultChefInstaller                          = "https://www.chef.io/chef/install.sh"
-	DefaultChefVersion                            = "15.5.17"
+	DefaultBootstrapInstaller                     = "https://www.chef.io/chef/install.sh"
```

Comment:
can we also remove these 2 defaults about chef & chef version?
and put additional info on readme
@fadlinurhasan

Reply:
Please take a look the changes mas https://github.com/pathfinder-cm/pathfinder-agent/pull/23/commits/9febc1817cc09f8e9d5611e8ee8932cc52da9d81